### PR TITLE
Sw 155 feedback for elbow sensor

### PR DESCRIPTION
**Pull request by SlowMiata**
created all the thresholds for each of the possible movements dumbbell and elbow
created a function to get the overall form for the workout
implemented the elbow feedback for every set
created dangerous form function (need to implement somehow still)

### CODE REVIEW

**Review by DanielEap — CHANGES_REQUESTED** [review]
Just change back SmartWeightsApp.swift to load the loginPage() 

Otherwise, everything looks great, remind us to have you demo this!

File: SmartWeights/SmartWeights/SmartWeightsApp.swift [hunk 1]
```diff
@@ -12,7 +12,8 @@ struct SmartWeightsApp: App {
     let persistenceController = PersistenceController.shared
     var body: some Scene {
         WindowGroup {
-            LoginView()
+            //bleView(ble: BLEcentral())
```
Comment (AdamRa97) [inline SmartWeights/SmartWeights/SmartWeightsApp.swift:15]: change this back to `LoginView()`

**Review by AdamRa97 — CHANGES_REQUESTED** [review]
once the changes are made i think it lgtm

**Comment by SlowMiata** [comment]
PLEASE DONT MERGE THIS PULL REQUEST YET

**Pull request closed without merging**